src/pages/Login.jsx

```diff
@@ -0,0 +1,28 @@
+import './login.css';
+
+const Login = () => (
+  <div className="form signup">
+    <div className="form-header">
+      <div className="show-signup">Sign Up</div>
+      <div className="show-signin">Sign In</div>
+      <div className="show-reset">Reset</div>
+    </div>
```

Comment:
반복되는 부분은 따로 컴포넌트화 하는게 좋습니다

또한 `nav`, `ul`, `li` 같은 tag를 이용하면 semantic하게 작성하면 더 좋을 거 같아요.
```suggestion
    <ul className="form-header">
      <li className="show-signup">Sign Up</li>
      <li className="show-signin">Sign In</li>
      <li className="show-reset">Reset</li>
    </ul>
```